associative-arrays.md

```diff
@@ -4,7 +4,7 @@
 хэш-таблицы. Ассоциативный массив с ключом типа `int` и значением типа `string`
```

Comment:
Yes, sorry for inattention, these examples are a bit messed up in English version. I fixed Russian part according to your change, thanks.